starters/express-apollo-prisma/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "endOfLine": "lf",
+  "printWidth": 100,
+  "singleQuote": true,
+  "tabWidth": 2,
```

Comment:
You can safely remove tab width here as it is redundant when using tabs for indentation.